source/index.md

```
@@ -23,9 +23,29 @@ docs/catalog
 
 # Snakemake workflow catalog
 
-<figure>
-    <img src="https://raw.githubusercontent.com/snakemake/snakemake.github.io/refs/heads/master/src/logo_light.svg"  alt="Snakemake workflow catalog" style="width:35%" />
-</figure>
+## News
+
+#### 2026-04-13
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)